types/context.go

```diff
@@ -3,44 +3,75 @@ package types
 import (
 	"bytes"
 	"fmt"
+	"github.com/ulule/loukoum/format"
```

Comment:
Could you add a newline here ?
Sorry to be nitpicking, it's just our internal code style at Ulule.

```go
import (
	"bytes"
	"fmt"

	"github.com/ulule/loukoum/format"
)
```